SSA/Projects/InstCombine/Tactic.lean

```diff
@@ -12,26 +12,46 @@ open MLIR AST
 open Std (BitVec)
 open Ctxt
 
+-- TODO: Do we call this dialect the "Alive" dialect (as hinted by the `simp_alive_peephole` name),
+--       or the "InstCombine" dialect (as hinted by the `InstCombine` namespace that everything
+--       lives in), or the "LLVM" dialect (as we do in the paper, and the simp-sets,
+--       and the `LLVM` namespace)?
```

Comment:
Should we make this an issue maybe, rather than dropping a TODO into the code base? The issue itself is very valid, but not very localized. I am afraid we may not find it very easily.

Reply:
Fair point: https://github.com/opencompl/ssa/issues/219